### PR TITLE
Dashboard Cards - Updated Ref. Impl. HTML/CSS

### PR DESCRIPTION
**Pull request by dtaylor113**
Now with filter dropdown in header or footer.

### CODE REVIEW

File: less/cards.less [hunk 1]
```diff
@@ -49,6 +45,10 @@
   }
 }
 
+.cards-pf {
```
Comment (rhamilto) [inline less/cards.less]: I'm not clear why you've moved this here as it introduces a mis-alphabetization.

Comment (dtaylor113) [inline less/cards.less]: Sorry, my bad, fixed

**Comment by dtaylor113** [comment]
Applied instructions for PR #134 "npm-shrinkwrap.json"

File: tests-src/_includes/cards.html [hunk 2]
```diff
@@ -76,6 +76,77 @@ <h2 class="card-pf-title">
...
+                    <ul class="dropdown-menu">
+                      <li><a role="menuitem">Filter 1</a></li>
+                      <li><a role="menuitem">Filter 2</a></li>
+                      <li><a role="menuitem">Filter 3</a></li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <span>card sub-title</span>
```
Comment (rhamilto) [inline tests-src/_includes/cards.html:103]: To improve html semantics, it'd be better to write this as &lt;h3 class="card-pf-subtitle"&gt;

**Comment by erundle** [comment]
Should we close this?

**Comment by dtaylor113** [comment]
Not yet, I need to meet with Robb and clean up my css

**Comment by erundle** [comment]
Yea sorry I was asking if this was still relevant. It sounds like it is.

**Pull request closed without merging**